includes/class-llms-db-ugrader.php

```diff
@@ -101,6 +101,37 @@ public function can_auto_update() {
 
 	}
 
+	/**
+	 * Retrieve the callback's prefix string based on the schema's namespace declaration.
+	 *
+	 * If `$namespace` is empty, no prefix will be added.
+	 * If `$namespace` is `true`, the namespace is assumed to be `LLMS\Updates`.
+	 * If `$namespace` is a string, that string will be used.
+	 *
+	 * If a namespace is found, `\Version_X_X_X` will automatically be appended to the namespace. The
+	 * string `X_X_X` is the database version for the upgrade substituting underscores for dots.     *
+	 *
+	 * @since [version]
+	 *
+	 * @param array  $info    Upgrade schema array.
+	 * @param string $version Version string for the upgrade.
+	 * @return string
+	 */
+	protected function get_callback_prefix( $info, $version ) {
+
+		if ( ! empty( $info['namespace'] ) ) {
+
+			$ver = explode( '-', $version ); // Drop prerelease data.
```

Comment:
So this "won't work" with pre-releases.
Like we can't use this for a db update from `1.0.0-beta1` to `1.0.0-beta2`.
Shall we note this in the method documentation?